packages/validator/docs/optional.md

```diff
@@ -0,0 +1,47 @@
+
+# Optional
+
+Some parameters in specification can be present but in case they are not there, it shouldn't be an issue as well. For this `__optional` is used, parameters inside it will not cause any extra parameter messages or missing parameter messages to show.
```

Comment:
Suggested:
> By default, parameters in the specification are required. However, there is the option of creating optional parameters, in case certain parameters are present but _not_ required.